MykhailKlymenkoHomework1.cpp

```diff
@@ -0,0 +1,74 @@
+#include <cmath>
+#include <iostream>
+
+int main() {
+  constexpr int minRange{1};
+  constexpr int maxRange{255};
+  constexpr int val1{4};
+  constexpr int val2{2};
+
+  float a{}, b{}, c{}, x{}, x1{}, x2{};
+
+  std::cout << "Quadratic equation looks like: a*x^2 + b*x + c = 0, where 'a' "
+               "doesn't equal 0 "
+            << '\n';
+
+  std::cout << "Please enter 'a' value in range [" << minRange << "..."
+            << maxRange << "]:\t";
+
+  std::cin >> a;
+
+  if (a > maxRange || a < minRange) {
+    std::cout << "Incorrect input a";
+    return -1;
+  }
+
+  std::cout << "Please enter 'b' value in range [" << minRange << "..."
+            << maxRange << "]:\t";
+
+  std::cin >> b;
+
+  if (b > maxRange || b < minRange) {
+    std::cout << "Incorrect input b\n";
+    return -1;
+  }
+
+  std::cout << "Please enter 'c' value in range [" << minRange << "..."
+            << maxRange << "]:\t";
+
+  std::cin >> c;
+
+  if (c > maxRange || c < minRange) {
+    std::cout << "Incorrect input c\n";
+    return -1;
+  }
+
+  float D = (b * b) - (val1 * a * c);
```

Comment:
```suggestion
  float D { (b * b) - (val1 * a * c) };
```
use uniform initialization